app.js

```diff
@@ -263,6 +263,18 @@ const emptyOverlay = () => {
   }
 };
 
+const shouldQuit = app.makeSingleInstance(() => {
+  // Someone tried to run a second instance, we should focus our window.
+  if (mainWindow) {
+    if (mainWindow.isMinimized()) mainWindow.restore();
+    mainWindow.focus();
+  }
+});
+
+if (shouldQuit) {
+  app.quit();
```

Comment:
The example includes a `return;` after this line. Is it needed?

Reply:
It is not.